### ISSUE TITLE
Bug report - Button push is registered twice on TZ3000_wkai4ga5

### ISSUE THREAD

**Issue by strejda603** [JohanBendz/com.tuya.zigbee#793]
Hello @JohanBendz,

first of all, thank you very much for your work on Tuya Zigbee App for Homey!

### Describe the bug

When pushing any button on 4 Gang Wall Remote (TZ3000_wkai4ga5), a light switch flow assigned to that button is run twice.

### To Reproduce

1. Create a flow (see screenshot)
2. Push any button on the actual remote
3. See error

### Expected behavior

After pushing a button, a flow should run only once.

### App and Homey version

* Homey Pro (Early 2023)
* FW v10.3.1
* Tuya Zigbee App v0.2.25 (test)

### Screenshots
![Screenshot](https://github.com/JohanBendz/com.tuya.zigbee/assets/32793325/616d91da-f077-4824-85f9-474c968196be)

### Additional context

[Small talk on Community Forum](https://community.homey.app/t/app-pro-tuya-zigbee-app/26439/4660)
[Reference to another issue](https://github.com/JohanBendz/com.tuya.zigbee/issues/457)

**Comment by pwvandeursen** [comment]
You need to change the setting of the unit from switch to push button mode. this can only be done in tbe Tuya gateway which you need to buy and connect the unit to

**Comment by strejda603** [comment]
> You need to change the setting of the unit from switch to push button mode. this can only be done in tbe Tuya gateway which you need to buy and connect the unit to

What? So you're telling me, that this 400$ device can't change mode of 20$ button and 25$ gateway can? 😮

**Comment by pwvandeursen** [comment]
Eeeuh yes, because it is not made to change all settings of all different devices of all different suppliers. Needs to be programed in. And again tuya support is not made by Homey but by Johan and all his supporting co-collaborators.

**Comment by strejda603** [comment]
When I try to use that switch when it's in "**Interview** mode", it works good without duplications...

**Comment by strejda603** [comment]
I have found a *"solution"* for this bug:

I've created a new numeric variables with value **4**, and then using [Better Logic Library](https://homey.app/en-cz/app/net.i-dev.betterlogic/Better-Logic-Library/) app in a flow (as described below), I was able to make buttons work good without purchasing any additional gateway and changing modes. If you're interested, you could probably somehow integrate it in the driver. 🙂

![Screenshot](https://github.com/JohanBendz/com.tuya.zigbee/assets/32793325/77e5560c-a4d8-4ce2-a658-026b7c0e0f5c)

**Comment by JohanBendz** [comment]
@strejda603 Problem solved in new version of the app. Check your flows.
